src/models/TestResultsSchema.js

```diff
@@ -52,8 +52,8 @@ const testTypesSchema = Joi.object().keys({
   numberOfSeatbeltsFitted: Joi.number().max(4).required(),
   lastSeatbeltInstallationCheckDate: Joi.date().required(),
   seatbeltInstallationCheckDate: Joi.boolean().required(),
-  testResult: Joi.any().only([ 'Failure', 'Pass', 'PRS', 'Abandonned', 'Successful', 'Unsuccessful' ]).required(),
-  prohibitionIssued: Joi.boolean().required(),
+  testResult: Joi.any().only([ 'failure', 'pass', 'prs', 'abandonned', 'successful', 'unsuccessful' ]).required(),
+  prohibitionIssued: Joi.any().only([ 'yes', 'no' ]).required(),
```

Comment:
As per spec, this has been changed to boolean. `Joi.boolean().required()`.

Reply:
fixed